posts/heatmaps-clustering/index.qmd

```diff
@@ -4,6 +4,7 @@ subtitle: "A note on clustering distance matrices before plotting. Think sns.clu
 date: "08/03/2022"
 image: "visualization.png"
 categories: [visualisation, how-to]
+bibliography: ../references.bib
```

Comment:
if we move this to _metadata.yml we wouldn't have to include in each post. metadata does just that - populates all posts with default meta. but not sure whether we will also have some posts where we want them to have their own bib file.